doc/user/content/sql/alter-cluster.md

```diff
@@ -6,44 +6,101 @@ menu:
     parent: 'commands'
 ---
 
-`ALTER CLUSTER` changes the configuration of a cluster, which **might incur
-downtime**. To rename a cluster, use [`ALTER ... RENAME`](/sql/alter-rename/).
+`ALTER CLUSTER` changes the configuration of a cluster, such as the `SIZE` or
+`REPLICATON FACTOR`.
+
+{{< admonition type="Disambiguation" >}}
+
+This page covers the usage of `ALTER CLUSTER` to change the cluster's
+*configuration*. To change the name of a cluster, see [`ALTER ... RENAME
+TO`](/sql/alter-rename/).
+
+{{</ admonition >}}
 
 ## Syntax
 
-#### Set new configuration value
+`ALTER CLUSTER` has the following syntax forms:
+
+- To set a cluster configuration:
+
+  ```mzsql
+  ALTER CLUSTER <cluster_name>
+  SET (
+    <config1> = <config_value1> [, ... ]  -- equal sign ('=') is optional
```

Comment:
Alternatively, we could list each config here as well as in the table below:

```
ALTER CLUSTER <cluster_name>
SET (
    SIZE = <text>,                                     -- Optional.
    REPLICATION FACTOR= <int>,                      -- Optional.
    INTROSPECTION INTERVAL = <interval>,     -- Optional.
    INTROSPECTION DEBUGGING = <bool>,      -- Optional.
    MANAGED = <bool>,                                         -- Optional.
    SCHEDULE = [MANUAL | ON REFRESH (...)]  -- Optional.
)
WITH ( <command_options> )                             -- Optional.
;
```

And later, in the command options section, flesh out the 

```
WITH (
      WAIT UNTIL READY ([TIMEOUT | ON TIMEOUT [COMMIT|ROLLBACK])
)

-- or --

WITH (
      WAIT FOR <duration> 
)
```

So, basically, we can do a lot of things.  So, don't be shy.

Reply:
So ... added all the options.  and went with using [ ] ... but for now, punting on using italics as non-trivial.

Reply:
Tweaked further because using [ ]  for all that could be optional is too messy and doesn't actually help people in practice since they would have to copy and remove all the extraneous [ ] , which is tedious.